src/test/scala/temple/RegenerationFilterTest.scala

```diff
@@ -1,68 +1,80 @@
 package temple
 
 import java.io.{File, PrintWriter}
+import java.nio.file.{Files, Path}
 
 import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
 import temple.builder.project.Project
 import temple.detail.PoliceSergeantNicholasAngel
-import temple.generate.FileSystem
+import temple.generate.{FileMatchers, FileSystem}
+import temple.utils.FileUtils
 
 import scala.reflect.io.Directory
 
-class RegenerationFilterTest extends FlatSpec with Matchers with BeforeAndAfter {
+class RegenerationFilterTest extends FlatSpec with FileMatchers with BeforeAndAfter {
 
   behavior of "RegenerationFilterTest"
 
-  private val testDirectory = "/tmp/test"
+  private var testDirectory: Path = Files.createTempDirectory("test")
```

Comment:
```suggestion
  private var testDirectory: Path = _
```

This seems like a pointless duplication of `before`